.github/workflows/build-be.yml

```
@@ -12,22 +12,23 @@ on:
 
 jobs:
   build:
+    name: Build and Publish Backend Container
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout Code
+      - name: Check Out Repository
         uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)